static/css/main.css

```diff
@@ -90,32 +45,100 @@ header {
   margin: 5px auto;
 }
 
-/*MEDIA QUERIES*/
 
-@media (max-width: 870px) {
-  header {
-    /*height: 100px;*/
-    padding: 2em .8em 8em;
-  }
 
-  #nav {
-    float: left;
-  }
+/*=======Members section==============*/
+
+.members{
+  background-color: rgb(51, 51, 51);
+}
+
+.members__heading{
+  color: #fff;
+  font-family: Verdana, sans-serif;
+  margin-top: 30px;
+}
+
+.member{
+  max-width:260px;
+  margin-top: 50px;
+  margin-bottom: 50px;
+  text-align: center;
+  padding: 20px 15px;
+  border: 1px solid transparent;
+  border-radius: 5px;
+}
 
-  #nav li a {
-    margin-top: .3em;
-  }
+.member:hover{
+  background-color: rgb(41, 41, 41);
+  border-radius: 5px;
+  -webkit-transition: all 0.3s linear 0s;
+  -moz-transition: all 0.3s linear 0s;
+  -o-transition: all 0.3s linear 0s;
+  transition: all 0.3s linear 0s;
+  border: 1px solid tomato;
 }
 
-@media (max-width: 600px) {
-  header {
-    /*height: 100px;*/
-    padding: 2em .8em 8.8em;
-  }
+.member__img{
+  height: auto;
+  max-width: 200px;
+  padding: auto auto;
+  -webkit-transition: all 0.3s linear 0s;
+  -moz-transition: all 0.3s linear 0s;
+  -o-transition: all 0.3s linear 0s;
+  transition: all 0.3s linear 0s;
+  overflow: hidden;
+  border-radius: 10px;
+  max-height:200px;
 
-  header h1 {
-    font-size: 1.4em;
-    margin-top: -20px;
-  }
 }
 
+.member__name{
+
+  font-family: Verdana, sans-serif;
+  color: rgb(237, 203, 51);
+
+}
+
+.member__role{
+  color: rgba(237, 203, 51, .8);
+  font-size: 16px;
+}
+
+.member__desc{
+  color: #fff;
+  font-family: Verdana, sans-serif;
+  display: block;
+  height: 240px;
```

Comment:
There should be a fixed height of member description, otherwise it would lead to different size of member profile and everything will look ugly